src/groups/mqb/mqbplug/mqbplug_plugintype.h

```diff
@@ -42,7 +42,7 @@ namespace mqbplug {
 /// authentication, ...)
 struct PluginType {
     // TYPES
-    enum Enum { e_STATS_CONSUMER };
+    enum Enum { e_STATS_CONSUMER = 0, e_AUTHENTICATOR = 1 };
```

Comment:
I would like to fix the first enum `e_STATS_CONSUMER` by removing S from the name: `e_STAT_CONSUMER`. We have `mqbblp_statconsumer.*` sources without S in name